src/sentry/api/endpoints/organization_missing_org_members.py

```diff
@@ -63,7 +63,7 @@ def _get_shared_email_domain(self, organization):
         # if a member has user_email=None, then they have yet to accept an invite
         org_owners = organization.get_members_with_org_roles(
             roles=[roles.get_top_dog().id]
-        ).exclude(user_email=None)
+        ).exclude(Q(user_email=None) | Q(user_email=""))
```

Comment:
Curious how could the email be an empty string? Do we allow them to not set it when accepting an invite or creating a org?

Reply:
i'm not sure. looking at the issue, the email looks like an empty string. maybe the `user_email` is an empty string by default rather than `None` in prod? locally i see it as `None`